docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -719,6 +720,21 @@ testcafe ie my-tests --page-load-timeout 0
 
 *Related configuration file property*: [pageLoadTimeout](configuration-file.md#pageloadtimeout).
 
+### --browser-init-timeout \<ms\>
+
+Specifies the time (in milliseconds) for every of the specified browsers to connect and report that it is ready to test. If one or more of the browsers fail to connect within the specified period, an error is thrown.
```

Comment:
> every of the specified browsers

Can't this be simplified to *every specified browser*?

Reply:
> one or more of the browsers

*one or more browsers* is also legit, isn't it?

Reply:
I thought about it. 

> Time (in milliseconds) for **all** browsers to connect to TestCafe and report that they are ready to test. If one or more of the browsers fail to connect within the specified period, an error is thrown.

OR

> Time (in milliseconds) for every browser to connect to TestCafe and report that it is ready to test. If one or more of the browsers fail to connect within the specified period, an error is thrown.

Is it obvious that we mean `all browsers you specified`, not `all browsers in the world`?